GitUI/GitUIExtensions.cs

```diff
@@ -112,24 +123,26 @@ void OpenWithDiffTool()
                         : diffOfConflict;
                 }
 
-                if (file.IsSubmodule
-                    && file.GetSubmoduleStatusAsync() is Task<GitSubmoduleStatus> task)
+                var task = file.GetSubmoduleStatusAsync();
```

Comment:
Newbie question: Does this really nothing yet if it is not a submodule?

Reply:
The task is only set if this is a submodule, so the first check is not technically needed (I prefer it though)

Reply:
There is an opportunity to improve this code pattern to make it clearer. It is a bit confusing as it stands.